packages/keyring-controller/src/KeyringController.ts

```diff
@@ -758,6 +783,15 @@ export class KeyringController extends BaseController<
     });
   }
 
+  async createKeyringFromMnemonic(mnemonic: string): Promise<string> {
```

Comment:
Why do clients need this function if they can use `addNewKeyring`? It would be nice to avoid new keyring-specific methods on `KeyringController` since we are already trying to get rid of the ones that are already in place (e.g. QR-related ones)